### DISCUSSION TITLE
Why is the .js code not formated in git ?

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#254] [Q&A] [answered]
hi, I try to get more into VUE. I'm wondering why the .js file are not formated in git and alsonot in vscode.
Is there a formated view optin in git ?
How can I format the code in vscode ?

**Comment by mgineer85** [comment, marked as answer]
It’s just the built result.
The sources are over here: https://github.com/photobooth-app/photobooth-frontend
